docs/knowledge_base/tutorials/understanding_gui/step_02/step_02.md

```diff
@@ -15,9 +15,8 @@ more within the graphical interface.
 
 ## Visual elements
 
-Taipy GUI can be considered as an **augmented** Markdown; it adds the concept of 
-**[Visual elements](../../../../manuals/gui/viselements/index.md)** on top of all the Markdown 
-syntax. A visual element is a Taipy graphical object displayed on the client. It can be a 
+When using the Mardown syntax, Taipy augments it with the concept of 
+**[visual elements](../../../../manuals/gui/viselements/index.md)**. A visual element is a Taipy graphical object displayed on the client. It can be a 
```

Comment:
```suggestion
**[visual elements](../../../../manuals/gui/viselements/index.md)**. A visual element is a
Taipy graphical object displayed on the client. It can be a 
```